src/relay/ir/expr_functor.cc

```diff
@@ -211,12 +365,12 @@ Expr ExprMutator::VisitExpr_(const MatchNode* m) {
   for (const Clause& p : m->clauses) {
     clauses.push_back(VisitClause(p));
   }
-  return Match(VisitExpr(m->data), clauses, m->complete);
```

Comment:
The changes from VisitExpr -> Mutate is no longer necessary, please change them back

Reply:
@tqchen Every other method in this class already uses mutate instead of VisitExpr, this makes ExprMutator more internally consistent, even if it's not strictly needed. I prefer it for that reason, but it doesn't need to be part of the PR, if you really want me to remove it.